styles.css

```diff
@@ -317,6 +409,11 @@ ul {
   color: #f5f5ff;
```

Comment:
### NB:
- [ ] I have realized from the commits that the project is a collaborative project of 3 right? If that is the case, kindly add the contributors to the author section of the README file

Reply:
@addod19 , Thanks for the feedback.
You're appreciated 🙃